spec/requests/transactions_spec.rb

```diff
@@ -4,6 +4,8 @@
       @content_item = content_store_has_example_item("/foo", schema: "transaction")
     end
 
+    it_behaves_like "it can render the govuk_chat promo banner", "/foo"
```

Comment:
Minor: The chat promo banner has been removed, so this example no longer exists.

Reply:
Fixed